contents/handbook/people/team-structure/session-recording/objectives.mdx

```diff
@@ -0,0 +1,13 @@
+### Objective: Start charging for session recordings resulting in a $100K increase in ARR from new customers
+
+#### KR1: Payment and billing flow for sessions recordings is in place ASAP
```

Comment:
This seems more like a task/activity than a measurable key result. How much increase in ARR do you expect this to generate?